stairway/src/main/java/bio/terra/stairway/impl/FlightRunner.java

```diff
@@ -144,11 +144,9 @@ private FlightStatus fly() throws InterruptedException {
       // the UNDO in the first place.
       flightDao.step(flightContext);
       logger.error(
-          "DISMAL FAILURE: non-retry-able error during undo. Flight: {}({}) Step: {}({})",
-          flightContext.getFlightId(),
-          flightContext.getFlightClassName(),
-          flightContext.getStepIndex(),
-          flightContext.getStepClassName());
+          "{} (index {}) experienced DISMAL FAILURE: non-retryable error on undo",
```

Comment:
Drive-by: now that we include flight ID and class on all flight logs in the MDC, we can trim down the dismal failure log line.

I also reformatted to lead with the failing step class based off of Stairway retro feedback that all of the useful information on a log line is at the end.  Do we like this better?  Unfortunately we don't get step context in MDC for free at this point -- we're outside of step execution.